frontend/src/components/Main/OptionsForm.js

```diff
@@ -0,0 +1,133 @@
+import React, {Component} from "react";
+import OptionsList from "./OptionsList";
+
+import {inject, observer} from "mobx-react";
+
+@inject("DataStore")
+@observer
+class OptionsForm extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            optionsList: [
+                {
+                    bmr_type: "",
+                    bmr_value: "",
+                    tail_probability: "",
+                    confidence_level: "",
+                    distribution: "",
+                    variance: "",
+                    polynomial_restriction: "",
+                    background: "",
+                },
+            ],
+        };
+    }
+
+    onChange = e => {
```

Comment:
could this be smarter? you could use `e.target.name` and `e.target.value` to just change the selected field